tools/eosshellcontent.py

```diff
@@ -51,6 +52,8 @@ class ShellContent:
 
     def __init__(self):
         self._translations, self._langs = self._get_translations_dict()
+        # Regex for matching anything starting with a <tag> like format
+        self._tag_expression = re.compile('^\s*\<\w+\>.*')
```

Comment:
Should you not check that it starts with `<p>` instead? I guess it's OK otherwise.